mappings/net/minecraft/data/server/BlockLootTableGenerator.mapping

```diff
@@ -1,67 +1,104 @@
 CLASS net/minecraft/class_2430 net/minecraft/data/server/BlockLootTableGenerator
-	FIELD field_11336 NEEDS_SILK_TOUCH Lnet/minecraft/class_4570$class_210;
-	FIELD field_11337 DOESNT_NEED_SILK_TOUCH Lnet/minecraft/class_4570$class_210;
-	FIELD field_11338 JUNGLE_SAPLING_DROP_CHANCES_FROM_LEAVES [F
-	FIELD field_11339 SAPLING_DROP_CHANCES_FROM_LEAVES [F
-	FIELD field_11340 ALWAYS_DROPPED_FROM_EXPLOSION Ljava/util/Set;
-	FIELD field_11341 DOESNT_NEED_SILK_TOUCH_SHEARS Lnet/minecraft/class_4570$class_210;
-	FIELD field_11342 NEEDS_SILK_TOUCH_SHEARS Lnet/minecraft/class_4570$class_210;
-	FIELD field_11343 NEEDS_SHEARS Lnet/minecraft/class_4570$class_210;
+	FIELD field_11336 WITH_SILK_TOUCH Lnet/minecraft/class_4570$class_210;
+	FIELD field_11337 WITHOUT_SILK_TOUCH Lnet/minecraft/class_4570$class_210;
+	FIELD field_11338 JUNGLE_SAPLING_DROP_CHANCE [F
+	FIELD field_11339 SAPLING_DROP_CHANCE [F
+	FIELD field_11340 EXPLOSION_IMMUNE Ljava/util/Set;
```

Comment:
I know this is a vanilla-inspired name, but `EXPLOSION_IMMUNE` is a little confusing imo, however I can't think of a better name either so I guess it's fine